rasa/core/agent.py

```diff
@@ -665,33 +665,6 @@ async def handle_text(
 
         return await self.handle_message(msg, message_preprocessor)
 
-    def _max_history(self) -> int:
```

Comment:
dropped that as agreed in https://rasa-hq.slack.com/archives/C8LEHMTAM/p1626107441347300?thread_ts=1625575547.183300&cid=C8LEHMTAM

Reply:
(could have done that later but it was in the way with a test)